src/service/viewer-impl.js

```diff
@@ -802,6 +806,18 @@ export class Viewer {
   }
 
   /**
+   * Retrieves the share-tracking identifier from the viewer
+   * @return {!Promise<string>}
+   */
+  getShareTrackingIncomingFragment() {
+    if (this.shareTrackingIncomingFragment_) {
```

Comment:
Lets check with `hasCapability` as to whether the viewer supports share tracking. You can make up the capability name.